notebooks/enterprise-search/app-search-engine-exporter.ipynb

```diff
@@ -620,20 +735,63 @@
     "result_fields = list(schema.keys())\n",
     "\n",
     "text_fields = [field_name for field_name in schema if schema[field_name] == \"text\"]\n",
-    "best_fields = [field_name + \".stem\" for field_name in text_fields]\n",
+    "best_fields = [field_name + \".stem\" for field_name in text_fields]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, from our text fields, we create set of fields with specified weights for our various analyzers.\n",
     "\n",
+    "* For the text field itself, we weight this as neutral, with a `1.0`\n",
+    "* For any stem fields, we weight this _slightly_ less to pull in closely stemmed words in the query.\n",
+    "* Any prefixes, we weight this with a minimal weight to ensure these do not dominate our scoring.\n",
+    "* For any potential bigram phrase matches, we weight these as well with a `0.75`\n",
+    "* Finally for our delimiter analyzed terms, we wight these somewhere in the middle.\n",
+    "\n",
+    "These are the default weightings that App Search uses. Feel free to experiement with these values to find a balance that works for you."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "cross_fields = []\n",
     "\n",
     "for text_field in text_fields:\n",
     "    cross_fields.append(text_field + \"^1.0\")\n",
     "    cross_fields.append(text_field + \".stem^0.95\")\n",
     "    cross_fields.append(text_field + \".prefix^0.1\")\n",
     "    cross_fields.append(text_field + \".joined^0.75\")\n",
-    "    cross_fields.append(text_field + \".delimiter^0.4\")\n",
+    "    cross_fields.append(text_field + \".delimiter^0.4\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we're ready to create our actual payload for our query. This is analagous to the query that App Search uses when querying.\n",
+    "\n",
+    "Within this query, we first set an organic query rule. This defines a boolean query under the hood that allows a match to be found and scored either in our cross fields we defined above, or in the \"best fields\" as defined.\n",
+    "\n",
+    "For the results, we sort on our score descending as the primary sort, with the document id as the secondary.\n",
+    "\n",
+    "We apply highlights to our results, request a return size of the top 10 hits, and for each hit, return the result fields."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "\n",
     "app_search_query_payload = {\n",
     "    \"query\": {\n",
-    "        \"rule_query\": {\n",
+    "        \"rule\": {\n",
```

Comment:
It would have been nice to use retrievers here, perhaps we could wrap this query under a standard retriever (or a rule retriever!) or even use RRF?